docs/networking/calico.md

```diff
@@ -55,37 +75,44 @@ To enable this mode in a cluster, add the following to the cluster spec:
     calico:
       crossSubnet: true
 ```
-In the case of AWS, EC2 instances have source/destination checks enabled by default.
-When you enable cross-subnet mode in kOps 1.19+, it is equivalent to:
+In the case of AWS, EC2 instances' ENIs have source/destination checks enabled by default.
+When you enable cross-subnet mode in kOps 1.19+, it is equivalent to either:
 ```yaml
   networking:
     calico:
       awsSrcDstCheck: Disable
       IPIPMode: CrossSubnet
 ```
-An IAM policy will be added to all nodes to allow Calico to execute `ec2:DescribeInstances` and `ec2:ModifyNetworkInterfaceAttribute`, as required when [awsSrcDstCheck](https://docs.projectcalico.org/reference/resources/felixconfig#spec) is set.
+or
+```yaml
+  networking:
+    calico:
+      awsSrcDstCheck: Disable
+      encapsulationMode: vxlan
+      IPIPMode: Never
```

Comment:
Still needed, I think this change deprecates it?

Reply:
I was just trying to show that with the VXLAN encapsulation mode, the effective value of "IPIPMode" becomes "Never," but I guess it doesn't really matter; the main point of these examples is to show that "awsSrcDstCheck" becomes "Disable" automatically.

I'll edit this section again.